lexique-oclock/BDD.md

```diff
@@ -25,11 +37,17 @@
 
 `Champ`
 
-`Relation:` One to One(1,1), One to Many(1, N) Many to One (N, 1) Many to Many (N, N)
+`Relation:` 
+
+One to One(1,1), One to Many(1, N) Many to One (N, 1) Many to Many (N, N)
```

Comment:
Relation : Lien logique qu'il existe entre deux ou plusieurs entités au sein d'une base de données. Ce lien peut être de différent types : one-to-one (1:1), one-to-many (1:n), many-to-one(n:1, l'autre côté de la many-to-one) ou many-to-many (n:n).